web/tools/manifest.json

```diff
@@ -20,17 +20,17 @@
     "src": "favicons/favicon-512x512.png",
     "sizes": "512x512"
   }],
-  "homepage_url": "https://integreat-app.de/",
+  "homepage_url": "",
   "related_applications": [
     {
-      "platform": "play",
-      "id": "tuerantuer.app.integreat",
-      "url": "https://play.google.com/store/apps/details?id=tuerantuer.app.integreat"
+      "platform": "",
+      "id": "",
+      "url": ""
```

Comment:
Maybe fill these empty string with something like: `"This is replaced by webpack.config.babel.js"`

Reply:
I thought if anythings not working with writing a value it would be better to just have an empty string there?

Reply:
Changed it, as you suggested